lib/inc/internal/facts/windows/ssh_resolver.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
This file is identical to the one in posix namespace. Can both me merge in a single file in parent directory?

Reply:
I have took it under consideration, but refactoring to have single namespace might introduce some problem. I would avoid modify existing posix code

Reply:
Managed to do so, no problems seen so far, but I'm honestly not happy about the end result. This, combined with the extraction of common code in base class, made an ugly diff (also noisy due to changing paths/namespaces).
@ciprianbadescu : Please have a look at the 2nd commit and let me know if this is what you had in mind. Other suggestions/approaches are very welcomed! 😄